packages/web/src/lib/wallectconnect-mode.ts

```diff
@@ -25,8 +24,8 @@ export class WallectConnectSession {
 	private readonly algodClient: algosdk.Algodv2;
 	wcAccounts: string[];
 
-	constructor(chain: ChainType, connector?: WalletConnect) {
-		this.algodClient = clientForChain(chain);
+	constructor(chain: string, connector?: WalletConnect) {
```

Comment:
why changing this?

Reply:
because when I checked the `algoexplorerAlgod` which accepts ChainType as a parameter, I realised that it should be a string (i.e the network type TestNet/MainNet) not an object, thus I updated it

Reply:
`ChainType` is an enum of strings.
I think the only problem is when a user will like to integrate his other chain in a wallet. I'm not sure if he can do it though.

Reply:
let's leave it as a it was. We will change it in https://www.pivotaltracker.com/story/show/181620989 anyway.

Reply:
okay